Add info to Sensors/SensorTable Help

jerryg2003 opened the pull request.

bobjacobsen commented on help/en/html/tools/Sensors.shtml:
```diff
@@ -53,42 +51,16 @@
 
-        <h2>Sensor Properties</h2>
-
-        <h3><a name="Debounce" id="Debounce"></a>Sensor
```
This is the Sensor page, so discussing the debounce facility here seems appropriate. Is there a reason it needed to be removed?

bobjacobsen commented on help/en/html/tools/Sensors.shtml:
```diff
@@ -157,51 +111,83 @@
+            
+        <h2><a name="Debounce" id="Debounce"></a>Sensor Debounce Delay</h2>
+
+        <p>The Sensor Debounce Delay facility is there to help prevent
```
The repositioning away from the discussion of sensors themselves, plus calling this a "faclity" makes this sound like something apart from the Sensors themselves.  It's not.

bobjacobsen commented on help/en/package/jmri/jmrit/beantable/SensorTable.shtml:
```diff
@@ -29,39 +30,43 @@
       between columns, and sort the rows by clicking the column title (a triangle will appear indicating the direction of the sort).
-      
-      <br>
-      <img src="images/SensorTableGraphic.png" height="187" width="307" align="right" alt="Graphic Sensor state column">
```
This seems to remove the graphic that was mentioned just above.

bobjacobsen commented on help/en/package/jmri/jmrit/beantable/SensorTable.shtml:
```diff
@@ -88,43 +93,106 @@
-        <p>The "Initial Sensor State" menu item lets you choose what state
-        Internal Sensors will display after startup.</p>
-        </dd>
+       <p>The word <strong>Defaults</Strong> appears on the window's command line.  Clicking it brings up two options:       
```
I'm not sure what "command line" means here.  Isn't it just that there's a Defaults menu on th window?

jerryg2003 commented:
Bob - 

I repositioned Debounce info because it seemed less like something a novice would need to know.  I can certainly move it back.

The graphic was moved so that it looks better (to me) on the displayed window.  As a result, it appears in a different place in the HTML.

What do you call the line in a window that has File, Edit, Help, etc?  I admit command line isn’t right.  Probably should be Menu bar?

jerryg2003 commented:
Bob -

Moved Debounce info up some, fixed “command line” to “menu bar,” some other things...FYI:  the debounce info was originally put in by @kevin-dickerson in 2011 when he called it a “facility.”  Based on your comment, I reworded to remove the word “facility.”

Jerry

bobjacobsen reviewed (approved):
Thanks!
